src/gameComponents/ScoreList.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="score-wrapper">
+    <b-table
+      class="table"
+      striped
+      hover
+      sticky-header="100%"
+      bordered
+      outlined
+      :items="scoreList"
+    >
```

Comment:
```suggestion
    />
```

Reply:
Thanks, edit it in [this commit](https://github.com/what-the-hub/vue-game/pull/37/commits/bb42cb19acff96dcf96195aa3545e743f4f2d779)